django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -463,7 +463,7 @@ input[type="submit"], button {
 
 /* Mobile */
 
-@media (max-width: 767px) {
+@media (max-width: 847px) {
```

Comment:
Changing the breakpoints is out-of-scope for the is ticket. It would need consideration as a (breaking?) change in a separate ticket. 

TBH I don't see the need: At 847px _Just close the sidebar_ seems a reasonable policy testing now.